Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# 30th Bug Crusade

## Changelog:
- Fixed an issue where you could keep the troops summoned via the offmap Old Gods' interface after a switch to a non-Old Gods' religion.
- Fixed an issue where Arthur Waycrest was dead in the 603 and 605 bookmarks. His wife, Meredith, ruled instead.
- Fixed an issue where Cho'gall kept "On a journey" trait after he changed his secret religion to non-Old Gods during his journey.
**_How to test:_** start as Cho'gall, wait for his events, embrace the Old Gods, when the event to go to Kalimdor appears, accept it and switch your religion to the Holy Light using commands mentioned below in the How to test.
- Fixed an issue where if both parents had Draenei, Vrykul or Dryad blood they didn't pass it to their children.
- Fixed an issue where Arik Scorpidsting had "Southern" culture in the 605 bookmark.
- Fixed an issue where Velraeeda and Lirath had adult portraits.
- Fixed an issue where Cyndia Hawkspear's gamepedia link leaded to a wrong page.
- Fixed an issue where Sevren, a ruler of Brill in 603, had no dynasty.
- Fixed an issue where new blood elven dynasties were named Smith.


## Script changelog (ignore if you can't read CK2 script):
- Reworked events in wc_dark_cult_events to use if's instead of break's.


# How to test:
Just a few console commands to help you:
- `religion old_gods_worship` to switch to Old Gods' religion.
- `add_offmap_currency offmap_cthun 10000` to get 10000 C'thun's currency.
- `add_offmap_currency offmap_nzoth 10000` to get 10000 N'zoth's currency.
- `add_offmap_currency offmap_yoggsaron 10000` to get 10000 Yogg-Saron's currency.
- `religion holy_light` to switch to the Holy Light religion AKA non-Old Gods' religion.
- `add_trait blood_vrykul <Character ID>` to give the Blood of Vrykul trait to the character.
- Check `pollinate` and `give_birth` usage [Here](https://ck2.paradoxwikis.com/Console_commands) to get children without waiting a few years.

Fixes #1430, fixes #1432, fixes #1434, fixes #1160, fixes #1476, fixes #1477, fixes #1454, dies #1472.
